float64 TX # yoke translation in X
float64 TY # yoke translation in Y
float64 TZ # yoke translation in Z
float64 RX # yoke rotation about X-axis
float64 RY # yoke rotation about Y-axis
float64 RZ # yoke rotation about Z-axis
float64 FX # yoke force exerted along X-axis
float64 FY # yoke force exerted along Y-axis
float64 FZ # yoke force exerted along Z-axis
float64 MX # yoke moment exerted about X-axis
float64 MY # yoke moment exerted about Y-axis
float64 MZ # yoke moment exerted about Z-axis